Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,45 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// A Boolean that is `true` if peer SSL certificates are to be verified.
+    ///
+    /// You can bypass verification by setting this property to `false` (not recommended).
+    public static var verifyPeerSSLCertificate = true
+    /// The URL representing a PEM-encoded certificate authority file to be used for
+    /// peer SSL certificate verification. Setting a certificate authority file takes
+    /// precedence over the value of verifyPeerSSLCertificate.
+    ///
+    /// See:
+    /// - https://curl.haxx.se/docs/sslcerts.html
+    /// - https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL? {
```

Comment:
My god, it’s a long story on Android…. very difficult to replicate in C. The simplest way is to add a raw asset for the cacerts.pem file then copy that to a place where you can get the actual file path in Java and pass that to Swift to set using this api. https://github.com/SwiftJava/swift-android-samples/blob/master/swifthello/src/main/java/net/zhuoweizhang/swifthello/SwiftHello.java#L35